README.md

```diff
@@ -39,12 +39,15 @@ If you want a simple guide to install Empirical Core, then you've come to the ri
 
 7. Install the bundle with `bundle install`.
 
+7. In the config folder, delete the ".example" extension from database.yml.example to read "database.yml"
```

Comment:
@regulus33 You asked about additional steps to configure `confg/database.yml`. Please take a look at step 13 on https://empirical-core.readme.io/docs/install-empirical-core-on-your-local-machine-linux

It currently reads as:
```
13. Create the database configuration file from the given example. Create
a user/password entry in the file (you will add this user/password pair
into PostgreSQL in the next step).
```